src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -2,15 +2,14 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Threading.Tasks;
 
 namespace Microsoft.R.Components.ConnectionManager {
-    public interface IConnection {
+    public interface IConnection: IConnectionInfo {
```

Comment:
They had different fields originally

---
In reply to: [77669938](https://github.com/Microsoft/RTVS/pull/2246#discussion_r77669938) [](ancestors = 77669938)